README.md

```diff
@@ -47,3 +47,41 @@ To stop and remove the containers:
 docker-compose down
 ```
 Note: Compose project names default to the name of the directory which contains the `docker-compose.yml` project file. You may wish to specify a project name using `--project-name` in order to give your images and running containers a more recognizable name.
+
+## Run iRODS Tests
+
+There are 3 main ways to run the iRODS test suite:
+ - Core tests: assumes CSP is the only server
+ - Topology tests: assumes 1 CSP and 3 CSCs
+ - Federation tests: assumes 2 federated CSPs
+
+In order to run the test suite against, for instance, Ubuntu 18.04 using a Postgres 10.12 database to host the catalog, run the following:
+```bash
+# --project-directory defaults to `pwd`
+cd projects/ubuntu-18.04/ubuntu-18.04-postgres-10.12
+# run tests using latest officially released packages
+python run_tests.py
```

Comment:
consider renaming `run_tests.py` to be... `run_core_tests.py`?

Does that... make everything horrible?

Then we'd have:
 - run_core_tests.py
 - run_topology_tests.py
 - run_federation_tests.py

Reply:
I don't have a problem with it.

Reply:
consider a better word than core...    but i don't have another suggestion...

single_server?  bah.

Reply:
I think this is good enough for now. This is the term being used in previous generations/colloquially within the team. Changing the name is not difficult (at this time)